content/product-engineering/engineering/code-graph/batch-changes/metrics.md

```diff
@@ -14,5 +14,6 @@ We track success metrics in a (private) [Looker dashboard](https://sourcegraph.l
 
 ## Definitions
 
-- **Batch Changes MAU:** A visitor who triggers any event happening on a Sourcegraph Batch Changes property in a given month. In practice, we do not track CLI events, so this effectively only includes events in the GUI, or uploading a spec.
-- **Batch Changes monthly contributor:** A user that has taken an action to create or manage a batch change in a given month. In practice, a user that has previewed, or applied or closed a batch change in a given month. New actions that are added in the future, such as [comment, merge, publish](https://sourcegraph.productboard.com/roadmap/2263724-batch-changes-releases/features/6775792/portal) will be added.
+- **Sourcegraph Batch Changes Property:** TBD
+- **Batch Changes MAU:** A [visitor](../../../../bizops/user_definitions.md#visitor) who triggers any GUI event happening on a Sourcegraph Batch Changes property in a given month. We do not yet track CLI events.
+- **Batch Changes monthly contributor:** An [active user](../../../../bizops/user_definitions.md#active-user) that has previewed, applied, or closed a batch change in a given month. New actions that are added in the future, such as [comment, merge, publish](https://sourcegraph.productboard.com/roadmap/2263724-batch-changes-releases/features/6775792/portal).
```

Comment:
- Is active user correct? Or do we mean registered user?
- Is the difference between this one and the above one just visitor vs. active user?

Reply:
A visitor used to be called a user, that was a rule set by BizOps a while ago. It
s very costly to update our telemetry systems away from that.

Reply:
@ebrodymoore what do you think is the right thing to do here?

Reply:
Malo is right - that definition of an active user has been used since the start of on-prem @ Sourecgraph as far as I know AND it would be costly to move away from. But I also think visitors are more of a Cloud term right now since we don't distinguish most of the user definitions on-prem like we do in Cloud

Reply:
@ebrodymoore in terms of documenting the difference or making this clear/match the definitions on the analytics definition page, what would you recommend? Is there some way we can disambiguate the usage?